build/print.go

```diff
@@ -121,7 +121,16 @@ func (p *printer) file(f *File) {
 	for i, stmt := range f.Stmt {
 		switch stmt := stmt.(type) {
 		case *CommentBlock:
-			// comments already handled
+			// This is necessary so that CommentBlock
```

Comment:
This does feel too long for 1 comment line, but maybe break into 2 instead of 3?